MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -3467,7 +3461,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\nxattr -cr ~/Library/Developer/Xcode/DerivedData || echo Clear\n";
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n# xattr -cr ~/Library/Developer/Xcode/DerivedData || echo Clear\n";
```

Comment:
Does it mean you commented this one out?

Reply:
Yes, we discussed this in the chat with the rest of the team, the conclusion was to disable this script since it slows the build a lot. If see tests start failing we can put it back. So far I didn't notice any failure.